docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,141 @@
+# How to integrate Autoware with your vehicle
+
+## 1. Prepare your real vehicle hardware
+
+Prerequisites for the vehicle:
+
+- An onboard computer that satisfies the Autoware Installation prerequisites (see [here](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/#prerequisites))
+- The following devices attached
+  - Drive-by-wire interface
+  - LiDAR
+  - Optional: Inertial measurement unit
+  - Optional: Camera
+  - Optional: GNSS
+
+## 2. Create maps
+
+You need both a pointcloud map and a vector map in order to use Autoware.
+
+### Create a pointcloud map
+
+Use third-party tools such as a LiDAR-based SLAM (simultaneous localization and mapping) package to create a pointcloud map in the `.pcd` format.
```

Comment:
Listing up some typical and concrete tools is better.
https://github.com/autowarefoundation/autoware/discussions/173

```
- [HDL](https://github.com/koide3/hdl_graph_slam)
- [LIO-SAM](https://github.com/TixiaoShan/LIO-SAM)
- ...
```